### PR TITLE
improved and fixed tests for company collections

### PR DESCRIPTION
**Pull request by feedmypixel**
- Fixed company collection tests
- Improved usage
- removed duplicate work

### Of note
On Dev leeloo you don't get search results back when searching for a company with spaces in its name. Sometimes you don't get results back at all. These tests have been written with mock data on my local and will pass once collection search results in dev leeloo have been fixed.

@jim68000 FYI ☝️ 

screen shot 2017-09-12 at 17 12 48

### CODE REVIEW

File: test/acceptance/features/collections-company.feature [hunk 1]
```diff
@@ -1,55 +1,45 @@
...
-Feature: View collection of company
+Feature: View company collections
   As an existing user
   I would like to view all the companies in one place
   And be able to read the company details as expected
+  And be able to navigate to a chosen company page
 
   Background:
-    Given I am an authenticated user on Data Hub website
+    Given I am an authenticated user on the Datahub website
```
Comment (teneightfive) [inline test/acceptance/features/collections-company.feature]: Is it called `the Datahub` now then? I thought Mills clarified it should just be called `data hub` all lowercase.

File: test/acceptance/features/collections-company.feature [hunk 2]
```diff
@@ -1,55 +1,45 @@
 @collections-company
-Feature: View collection of company
+Feature: View company collections
```
Comment (teneightfive) [inline test/acceptance/features/collections-company.feature]: `collection` singular? It's only testing the one collection.

File: test/acceptance/features/collections-company.feature [hunk 3]
```diff
@@ -1,55 +1,45 @@
...
+    Given I am an authenticated user on the Datahub website
 
   @collections-company-name
-  Scenario: View company name under collections
+  Scenario: View company name in collection results
 
-     When I create a new company
-     When I search for this company name
-     Then I see company name in the list
+    When I create a "UK private or public limited company"
```
Comment (teneightfive) [inline test/acceptance/features/collections-company.feature]: Could this `When` also be moved to `Background` as it seems to be being called in each scenario.

File: test/acceptance/features/collections-company.feature (hunk 1 above)
Comment (feedmypixel) [reply test/acceptance/features/collections-company.feature]: Ah yes was going to ask mills but forgot and presumed :) - thanks

Comment (teneightfive) [reply test/acceptance/features/collections-company.feature]: I was even wondering if the `on the data hub website` is too much info. Isn't just saying `I am an authenticated user` enough? We don't have multiple places where a user is authenticated.

**Pull request merged**